src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,176 @@
+= Contributing to the Koji developer site
+:page-slug: contribute-koji-developers
+:page-banner: {imagesDir}/example.png
+
+The Koji for Developers documentation site is built and maintained by developers just like you.
+You too can help by contributing to the site *structure*, writing new *articles*, or *suggesting edits* to existing documentation.
+
+Anyone can open an issue about documentation, or contribute a change with a pull request to the `madewithkoji/docs` https://github.com/madewithkoji/docs[GitHub repository].
+
+link to code of conduct?
+
+include::../_includes/tip-discord-server.adoc[tags=*]
+
+== Editing the documentation
+
+The documentation is written in https://asciidoc.org/[Asciidoc] which is a lightweight markup language for simplifying the creation of documentation.
+All the AsciiDoc files that map to this site can be found in the repository under `src/asciidoc/`.
+
+Here's a quick example:
+
+[source, asciidoc]
+----
+= Koji for Developers
+Create your own templates for https://withkoji.com[Koji].
+
+
+== Use any framework
+
+* React
+* Vanilla JS
+
+[source,javascript]
+console.log('Hello World');
+----
+
+
+To see more and get a better understanding of the syntax visit the https://powerman.name/doc/asciidoc[AsciiDoc cheatsheet] and the https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference].
+
+We also have are a few extensions on AsciiDoc syntax added in to make the developer site ever better.
+
+=== Page Attributes
+Page attributes are used to add extra data related to a document.
+Add these to the top of your document, just under the document title.
+
+Page slug::
+	This corresponds to the page url and is used for linking between different pages. +
```

Comment:
```suggestion
	Corresponds to the page URL and is used for linking between different pages. +
```
Also, re: your question about explaining the structure, we might want to mention `nav.json` and the `_includes` directory (not here, but we might link to the nav.json info from here)